test/parallel/taskPar/sungeun/private.chpl

```diff
@@ -24,22 +24,22 @@ class localePrivateData {
   type myStuff;
   // assumes maxTaskPar is the same across all locales
   const numTasks = if dataParTasksPerLocale==0 then here.maxTaskPar
-    else dataParTasksPerLocale;
+  else dataParTasksPerLocale;
   var slot: sync bool;
   var r = {0..#numTasks};
-  var temps: [r] myStuff;
+  var temps: [r] myStuff;  // TODO: initializer
```

Comment:
@bradcray - here is another test TODO comment added. Not sure what the TODO would be or if we should to it in this PR.